src/api/cow-subgraph/graphql.ts

```diff
@@ -2641,6 +2663,135 @@ export enum UniswapPool_OrderBy {
+  name_gt?: InputMaybe<Scalars['String']>;
+  name_gte?: InputMaybe<Scalars['String']>;
+  name_in?: InputMaybe<Array<Scalars['String']>>;
+  name_lt?: InputMaybe<Scalars['String']>;
+  name_lte?: InputMaybe<Scalars['String']>;
+  name_not?: InputMaybe<Scalars['String']>;
+  name_not_contains?: InputMaybe<Scalars['String']>;
+  name_not_contains_nocase?: InputMaybe<Scalars['String']>;
+  name_not_ends_with?: InputMaybe<Scalars['String']>;
+  name_not_ends_with_nocase?: InputMaybe<Scalars['String']>;
+  name_not_in?: InputMaybe<Array<Scalars['String']>>;
+  name_not_starts_with?: InputMaybe<Scalars['String']>;
+  name_not_starts_with_nocase?: InputMaybe<Scalars['String']>;
+  name_starts_with?: InputMaybe<Scalars['String']>;
+  name_starts_with_nocase?: InputMaybe<Scalars['String']>;
+  priceEth?: InputMaybe<Scalars['BigDecimal']>;
+  priceEth_gt?: InputMaybe<Scalars['BigDecimal']>;
+  priceEth_gte?: InputMaybe<Scalars['BigDecimal']>;
+  priceEth_in?: InputMaybe<Array<Scalars['BigDecimal']>>;
+  priceEth_lt?: InputMaybe<Scalars['BigDecimal']>;
+  priceEth_lte?: InputMaybe<Scalars['BigDecimal']>;
+  priceEth_not?: InputMaybe<Scalars['BigDecimal']>;
+  priceEth_not_in?: InputMaybe<Array<Scalars['BigDecimal']>>;
+  priceUsd?: InputMaybe<Scalars['BigDecimal']>;
+  priceUsd_gt?: InputMaybe<Scalars['BigDecimal']>;
+  priceUsd_gte?: InputMaybe<Scalars['BigDecimal']>;
+  priceUsd_in?: InputMaybe<Array<Scalars['BigDecimal']>>;
+  priceUsd_lt?: InputMaybe<Scalars['BigDecimal']>;
+  priceUsd_lte?: InputMaybe<Scalars['BigDecimal']>;
+  priceUsd_not?: InputMaybe<Scalars['BigDecimal']>;
+  priceUsd_not_in?: InputMaybe<Array<Scalars['BigDecimal']>>;
+  symbol?: InputMaybe<Scalars['String']>;
+  symbol_contains?: InputMaybe<Scalars['String']>;
+  symbol_contains_nocase?: InputMaybe<Scalars['String']>;
+  symbol_ends_with?: InputMaybe<Scalars['String']>;
+  symbol_ends_with_nocase?: InputMaybe<Scalars['String']>;
+  symbol_gt?: InputMaybe<Scalars['String']>;
+  symbol_gte?: InputMaybe<Scalars['String']>;
+  symbol_in?: InputMaybe<Array<Scalars['String']>>;
+  symbol_lt?: InputMaybe<Scalars['String']>;
+  symbol_lte?: InputMaybe<Scalars['String']>;
+  symbol_not?: InputMaybe<Scalars['String']>;
+  symbol_not_contains?: InputMaybe<Scalars['String']>;
+  symbol_not_contains_nocase?: InputMaybe<Scalars['String']>;
+  symbol_not_ends_with?: InputMaybe<Scalars['String']>;
+  symbol_not_ends_with_nocase?: InputMaybe<Scalars['String']>;
+  symbol_not_in?: InputMaybe<Array<Scalars['String']>>;
+  symbol_not_starts_with?: InputMaybe<Scalars['String']>;
+  symbol_not_starts_with_nocase?: InputMaybe<Scalars['String']>;
+  symbol_starts_with?: InputMaybe<Scalars['String']>;
+  symbol_starts_with_nocase?: InputMaybe<Scalars['String']>;
+};
+
+export enum UniswapToken_OrderBy {
+  Address = 'address',
+  AllowedPools = 'allowedPools',
+  Decimals = 'decimals',
+  Id = 'id',
+  Name = 'name',
+  PriceEth = 'priceEth',
+  PriceUsd = 'priceUsd',
+  Symbol = 'symbol'
+}
+
```

Comment:
All these seems unrelated, although we can relax it and add it in this PR already